man/integritytab.xml

```diff
@@ -72,6 +72,16 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>no-journal</option></term>
+
+        <listitem><para>
+        Disable the journal. Correspods to the "direct writes" mode documented in Documentation/admin-guide/device-mapper/dm-integrity.rst. Note that
```

Comment:
"Corresponds", and add a link to the upstream document via https://docs.kernel.org/

Reply:
Done